README.md

```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
 - Run ``npm run serve`` and open the displayed URL in your browser
 
 
+## Project Setup
+
+1. Clone the project from GitHub: `git clone <repository_url>`
+2. Open the project in Visual Studio Code.
+3. Deploy the contract.
+
+## Install Dependencies
+
+In the terminal of the Visual Studio Code console, follow these steps:
+
+1. Install Vue CLI globally: `npm install -g @vue/cli`
+2. Install web3: `npm install web3`
```

Comment:
Packages that are not installed globally should be installed through `npm install` via the `packages.json` file

Edit: web3 and big-integer are already covered in the `packages.json`, so instructions 2. and 3. are unnecessary, and instruction 1. is already in the **Prerequisites** section.